src/app/components/Pagination/index.tsx

```diff
@@ -46,16 +46,16 @@ interface PaginationProps {
 
 const visibilityToMediaQuery = (visibility: string) =>
   ({
-    [VISIBILITY.MOBILE_ONLY]: `display: none; @media (min-width: ${GROUP_2_MIN_WIDTH_BP}) and (max-width: ${GROUP_2_MAX_WIDTH_BP}) {
+    [VISIBILITY.MOBILE_ONLY]: `display: none; @media (min-width: ${GROUP_2_MIN_WIDTH_BP}rem) and (max-width: ${GROUP_2_MAX_WIDTH_BP}rem) {
```

Comment:
Do we want to use the shortcut media queries here - the _BP variables were not really meant to used across the application?

```suggestion
    [VISIBILITY.MOBILE_ONLY]: `display: none; ${GROUP_2_ONLY} {
```

Reply:
I tried doing this and it didn't work 😭 Still had numbers missing. It was hard to tell what exactly was working because sometimes things would work, sometimes they wouldn't - I thought maybe it wasn't updating properly so I would restart after each time to make sure, but the only reliable fix was the one I have in now, which always works.

Reply:
Ah, didn't realise that - thanks for letting me know. :shipit:  :shipit:  :shipit:  :shipit: